### PR TITLE
Add Steel Brain

### PR DESCRIPTION
**Pull request by steelbrain**
@sindresorhus I really like your idea of AMA :wink:

[Review on Reviewable](https://reviewable.io/reviews/sindresorhus/amas/105)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -93,6 +93,7 @@
 - [Arnaud Benard](https://github.com/arnaudbenard/ama) - Freelance JS developer and digital nomad.
 - [Anthony Ramella](https://github.com/tonyynot/ama) - I went from zero to hired web developer in 5 months. I [blog](http://anthonyramella.com) about my experiences learning code and working as a junior developer.
 - [Diki Andriansyah](https://github.com/dikiaap/ama) - Freelance full stack developer. Laravel addict.
+- [Steel Brain](https://github.com/steelbrain/ama) - The Rebel™
```
Comment (sindresorhus) [inline readme.md]: Lol, maybe add a short sentence of who you are?

Comment (sindresorhus) [inline readme.md]: Also end it with a `.`

Comment (steelbrain) [inline readme.md]: Done

**Comment by sindresorhus** [comment]
Welcome! :)

Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). Especially the ones closer to the bottom of the list. I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=

**Pull request closed without merging**